src/test/java/herokuapp/AddDeleteElementsTests.java

```diff
@@ -0,0 +1,61 @@
+package herokuapp;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class AddDeleteElementsTests {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test
+    public void addElementTest() {
```

Comment:
There should be a single test with following steps:
1. Add 2 elements
2. Delete one element
3. Verify elements count

Reply:
These 2 are redundant: addElementTest, deleteElementTest.
So please delete them